packages/core/src/__tests__/caip10-link.test.ts

```diff
@@ -108,94 +94,67 @@ describe('Ceramic API', () => {
     })
 
     it('Create and link DID', async () => {
-      const account = 'eip155:1:0x0544DcF4fcE959C6C4F3b7530190cB5E1BD67Cb7'
-      const linkProof = { account, did: ceramic.did.id }
-      authProvider.createLink.mockReturnValueOnce(linkProof)
+      authProvider.createLink.mockReturnValueOnce(PROOF)
 
-      const link = await Caip10Link.fromAccount(ceramic, account)
-      await link.setDid(ceramic.did, authProvider, { anchor: false })
+      const link = await Caip10Link.fromAccount(ceramic, BLOCKCHAIN_ACCOUNT)
+      await link.setDid(DID_USED, authProvider, { anchor: false })
 
-      expect(link.did).toEqual(ceramic.did.id)
+      expect(link.did).toEqual(DID_USED)
       expect(link.state.log).toHaveLength(2)
       expect(authProvider.createLink).toHaveBeenCalledTimes(1)
-      expect(validateLink).toHaveBeenCalledTimes(1)
-      expect(link.state).toMatchSnapshot()
-    })
-
-    it('Create and link DID with legacy CAIP format', async () => {
-      validateLink.mockClear()
-
-      const account = '0x0544DcF4fcE959C6C4F3b7530190cB5E1BD67Cb7@eip155:1'
-      const linkProof = { account, did: ceramic.did.id }
-      authProvider.createLink.mockReturnValueOnce(linkProof)
-
-      const link = await Caip10Link.fromAccount(ceramic, account)
-      await link.setDid(ceramic.did, authProvider, { anchor: false })
-
-      expect(link.did).toEqual(ceramic.did.id)
-      expect(link.state.log).toHaveLength(2)
-      expect(authProvider.createLink).toHaveBeenCalledTimes(1)
-      expect(validateLink).toHaveBeenCalledTimes(1)
```

Comment:
Why removed?

Reply:
I think, it is a remnant of merging `develop` into 2.x release branch at some point. I remember catching version change, but this one I did not see.

Reply:
seems important to make sure we keep support for the new caip10 format. Can we add the test back?

Reply:
But, no, this theory does not stand. I do not know.

Reply:
Yup, adding it back.

Reply:
Recovered.